src/bundle/Form/Data/TranslationAddData.php

```diff
@@ -15,13 +15,13 @@ class TranslationAddData extends BaseTranslationAddData
     /** @var string|bool */
     protected $translatorAlias;
 
-    /** @return mixed */
+    /** @return string|bool */
```

Comment:
why bool?

Reply:
looks like the form type for translation can be [checkbox](https://github.com/ibexa/automated-translation/blob/main/src/bundle/Form/Extension/TranslationAddType.php#L58) which will result in bool or [choice](https://github.com/ibexa/automated-translation/blob/main/src/bundle/Form/Extension/TranslationAddType.php#L74) which will results in string - depends on how many translation client providers was registered